src/App.tsx

```diff
@@ -83,11 +69,7 @@ export const App: React.FC = () => {
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
```

Comment:
pls [see falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
                {author && loaded && !hasError && !!posts.length && (
```
also pls always move such long expressions to a separate const